src/lesson7/same_name/Employee.java

```diff
@@ -0,0 +1,5 @@
+package lesson7.same_name;
```

Comment:
как мы обсуждали на уроке, сложные имена пакетов разделяются точкой или сложные выражения пишутся слитно.
```suggestion
package lesson7.same.name;
```